### PR TITLE
`VGP assembly pipeline`: some textual changes

### PR DESCRIPTION
**Pull request by abueg**
hello! 👋🏼 

- added disclaimer in intro paragraph about some results having minor differences depending on software versions
- fixed misleading sentence about the merqury plots
- specified tool version in body of text so users know the tutorial is referring to an older version (only done for `compute expression` as the newer version has different options, and for `bionano` as newer version was giving some errors we haven't had a chance to look into yet)

### CODE REVIEW

File: topics/assembly/tutorials/vgp_genome_assembly/tutorial.md [hunk 1]
```diff
@@ -824,7 +824,7 @@ The *Bionano Hybrid Scaffold* tool automates the scaffolding process, which incl
 
 > <hands-on-title>Bionano hybrid scaffolding</hands-on-title>
 >
-> 1. {% tool [Bionano Hybrid Scaffold](toolshed.g2.bx.psu.edu/repos/bgruening/bionano_scaffold/bionano_scaffold/3.6.1+galaxy2) %} with the following parameters:
+> 1. {% tool [Bionano Hybrid Scaffold](toolshed.g2.bx.psu.edu/repos/bgruening/bionano_scaffold/bionano_scaffold/3.6.1+galaxy2) %} (**VERSION 3.7+GALAXY0**) with the following parameters:
```
Comment (bgruening) [inline topics/assembly/tutorials/vgp_genome_assembly/tutorial.md:827]: Why this older version?

Comment (abueg) [reply topics/assembly/tutorials/vgp_genome_assembly/tutorial.md:827]: ah, sorry -- when Guido was testing the tutorial he said the newer version didn't work, so I put this edit to use the older version that we have tested before, but he updated me and said that the newer version did work eventually, so we can use the newer version! i'll take this note out

Comment (bgruening) [reply topics/assembly/tutorials/vgp_genome_assembly/tutorial.md]: I see, we fixed that recently.

**Review by bgruening — APPROVED** [review]

**Comment by bgruening** [comment]
Good luck.

**Comment by abueg** [comment]
thank you for the merge!!

**Pull request merged**